docs/docs/concepts/features/actions_v2.md

```diff
@@ -10,6 +10,11 @@ This is useful when you have special business requirements that ZITADEL doesn't
 We're working on Actions continuously. In the [roadmap](https://zitadel.com/roadmap), you see how we are planning to expand and improve it. Please tell us about your needs and help us prioritize further fixes and features.
 :::
 
+:::warning
+To use Actions v2 activate the feature flag "Actions" [feature flag](/docs/apis/resources/feature_service_v2/feature-service-set-instance-features, to be able to manage the related resources.
+The Actions v2 will always be executed if available, even if the feature flag is switched off, to remove any Actions v2 the related Execution has to be removed.
```

Comment:
@hifabienne Do we want to have actions v2 to be behind a feature flag? At what point do we delcare them on by default?

Reply:
Ok let's do it like that